tests/unit/Certify.spec.ts

```diff
@@ -118,6 +120,13 @@ describe('Certify', () => {
     expect(getLastEvent(wrapper, 'valid')).toBe(true)
   })
 
+  it('is valid when both certifiedBy, isCertified, and is staff are defined', () => {
+    const wrapper: Wrapper<Certify> = createComponent(trimmedCertifier, true, true)
+
+    // The last "valid" event should indicate that the form is valid.
+    expect(getLastEvent(wrapper, 'valid')).toBe(true)
+  })
+
```

Comment:
How about a test that it's invalid when staff is True but the other 2 values aren't True?

Reply:
Done